diff --git a/node_modules/@spaceribs/nx-web-ext/src/executors/serve/executor.js b/node_modules/@spaceribs/nx-web-ext/src/executors/serve/executor.js
index 1583ab9..d01d600 100644
--- a/node_modules/@spaceribs/nx-web-ext/src/executors/serve/executor.js
+++ b/node_modules/@spaceribs/nx-web-ext/src/executors/serve/executor.js
@@ -30,7 +30,7 @@ const runServe = (options, context) => tslib_1.__awaiter(void 0, void 0, void 0,
             const s = _c;
             if (s.success === true) {
                 devkit_1.logger.info('Application built successfully.');
-                const outputPath = s.outputPath || s.outfile.replace('main.js', '');
+                const outputPath = s.outputPath || s.outfile?.replace('main.js', '') || options.outputPath;
                 if (runnerSub == null) {
                     runnerSub = startWebExt(options.webExtOptions, outputPath).subscribe(() => {
                         devkit_1.logger.info('WebExt started successfully.');
