diff --git a/node_modules/nx/bin/run-executor.js b/node_modules/nx/bin/run-executor.js
index cf71d88d51823473cefe49e0382f6e8fd9b2dc3f..3de9a03e2ae3fc2b38465fcd63dd563090d86fc5 100644
--- a/node_modules/nx/bin/run-executor.js
+++ b/node_modules/nx/bin/run-executor.js
@@ -55,6 +55,7 @@ function setUpOutputWatching(captureStderr, streamOutput) {
     });
 }
 process.on('message', async (message) => {
+    process.env.SHARD = message.overrides.shard || '';
     try {
         const statusCode = await (0, run_1.run)(process.cwd(), process.env.NX_WORKSPACE_ROOT, message.targetDescription, message.overrides, message.isVerbose, message.taskGraph);
         process.exit(statusCode);
diff --git a/node_modules/nx/src/command-line/format/format.js b/node_modules/nx/src/command-line/format/format.js
index 4691116c22aee06970586b57fdd4140d95098789..8f94d10d715f2e7718ca8fa82fc7fa51c64faf5a 100644
--- a/node_modules/nx/src/command-line/format/format.js
+++ b/node_modules/nx/src/command-line/format/format.js
@@ -51,8 +51,8 @@ async function getPatterns(args) {
             return getPatternsFromProjects(args.projects, graph);
         }
         const p = (0, command_line_utils_1.parseFiles)(args);
-        const supportedExtensions = prettier
-            .getSupportInfo()
+        const supportedExtensions = (await prettier
+            .getSupportInfo())
             .languages.flatMap((language) => language.extensions)
             .filter((extension) => !!extension)
             // Prettier supports ".swcrc" as a file instead of an extension
